.github/workflows/ci.yml

```
@@ -52,11 +52,34 @@ jobs:
     steps:
       - name: Prepare OS
         run: |
-          (apt-get update && apt-get install curl -y) || true
-          yum install tar gzip -y || true
+          (apt-get update && apt-get install file curl -y) || true
+          yum install file tar gzip -y || true
       - uses: actions/checkout@v3
       - name: Run installer.sh
         run: sh docs/.vuepress/public/installer.sh
+      - name: Check kumactl architecture
+        run: if file ./kuma-*/bin/kumactl | grep x86-64 &> /dev/null; then exit 0; else exit 1; fi
+  installer-sh-arm64:
+    runs-on: ubuntu-latest
+    name: Test installer.sh on arm64
+    steps:
+      - uses: actions/checkout@v3
+      - uses: uraimo/run-on-arch-action@v2
+        name: Run commands
+        id: runcmd
+        with:
+          arch: aarch64
+          distro: ubuntu22.04
+
+          # Not required, but speeds up builds by storing container images in
+          # a GitHub package registry.
+          githubToken: ${{ github.token }}
+
+          # Set an output parameter `uname` for use in subsequent steps
+          run: |
+            apt-get update && apt-get install -y file curl tar gzip
+            sh docs/.vuepress/public/installer.sh
+            if file ./kuma-*/bin/kumactl | grep aarch64 &> /dev/null; then exit 0; else exit 1; fi
```

```suggestion
            file ./kuma-*/bin/kumactl | grep aarch64 &> /dev/null
```

Since it looks like `errexit` and `pipefail` are set here